src/app/beer_garden/api/http/handlers/v1/forward.py

```diff
@@ -37,7 +38,7 @@ async def post(self):
             schema:
                 $ref: '#/definitions/Forward'
         responses:
-          200:
+          204:
```

Comment:
Not related to this PR, but I noticed that this was wrong as the endpoint actually returns a 204.  At least in the one code path that appears to actually work.